BackofficeBundle/Form/Type/NodeType.php

```diff
@@ -121,6 +121,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
             ));
         if(!array_key_exists('disabled', $options) || $options['disabled'] === false){
+            $builder->add('templateSelection', 'collection', array(
```

Comment:
If you want to add a field in the form which is not mapped to the entity, you can use the mapped => false option